python/lsst/sims/catUtils/mixins/sncat.py

```diff
@@ -194,11 +248,14 @@ def get_snbrightness(self):
         if isinstance(bandname, list):
             raise ValueError('bandname expected to be string, but is list\n')
         bandpass = self.lsstBandpassDict[bandname]
+        # Adding photometric parameters
+        _photParams = PhotometricParameters()
```

Comment:
The `PhotometryBase` mixin (and, by extension, `PhotometryStars` and `PhotometryGalaxies`) defines a `self.photParams` member variable that users can change if they see fit.  I am worried that someone might instantiate a galaxy or star catalog with a custom `photParams` and then assign it to a supernova catalog and assume that everything was self consistent.  Can you, instead of creating `_photParams` inside of this method, do something like:

```
if not hasattr(self, 'photParams'):
    self.photParams = PhotometricParameters()
```
and then use `self.photParams` in the rest of the code?